__tests__/App.js

```diff
@@ -1,10 +1,11 @@
-import 'react-native';
-import React from 'react';
-import App from '../App.tsx';
+import 'react-native'
+import React from 'react'
+import renderer from 'react-test-renderer'
+import App from '../src/App.tsx'
 
 // Note: test renderer must be required after react-native.
-import renderer from 'react-test-renderer';
 
+// eslint-disable-next-line no-undef
```

Comment:
Why are you ignore eslint validation on this line ?

Reply:
Because that will be the only place in our code where our function will not be defined before being called.